python/lsst/ip/isr/ampOffset.py

```diff
@@ -457,8 +485,9 @@ def getInterfaceOffset(self, ampIdA, ampIdB, edgeA, edgeB):
             interfaceOffset = 0
             self.log.warning(
                 f"The fraction of unmasked pixels for amp interface {interfaceId} is below the threshold "
-                f"({self.config.ampEdgeMinFrac}) or the absolute offset value exceeds the limit "
-                f"({self.config.ampEdgeMaxOffset} ADU). Setting the interface offset to 0."
+                f"({ampEdgeGoodFrac:.2f} <? {self.config.ampEdgeMinFrac}) or the absolute offset value "
+                f"exceeds the limit ({np.abs(interfaceOffset):.2f} >? {self.config.ampEdgeMaxOffset} ADU). "
```

Comment:
I don't understand the `<?` and `>?` syntax here - what exactly are you trying to say?